lib/src/collections.dart

```diff
@@ -0,0 +1,18 @@
+import 'package:typesense/src/api_call.dart';
+
+class Collections {
+  ApiCall _apicall;
+  static const String RESOURCEPATH = '/collections';
+
+  Collections(apicall) {
+    this._apicall = apicall;
+  }
+
+  retrieve() {
+    return this._apicall.get(Collections.RESOURCEPATH);
+  }
+
+  create(schema) {
```

Comment:
```dart
Future<Map<String, dynamic>> create(Map<String, dynamic> schema) async {
  return await ....
}

Reply:
Since you have suggested using static datatypes. I will also update these in `retrieve()` function also